angular/src/app/landing/sections/resourceidentity.component.spec.ts

```diff
@@ -13,7 +13,7 @@ import { AuthService, WebAuthService, MockAuthService } from '../editcontrol/aut
 import { GoogleAnalyticsService } from '../../shared/ga-service/google-analytics.service';
 import { config, testdata } from '../../../environments/environment';
 
-describe('ResourceIdentityComponent', () => {
+fdescribe('ResourceIdentityComponent', () => {
```

Comment:
Will fdescribe() cause other unit tests disabled?

Reply:
Yes, thank you for catching that.